ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -1046,86 +1046,66 @@ def test024_archive_sub_order(self):
 
         self.assertEqual(len(rows_count) - 1, 1)
 
-    # will continue with us
-    @skip('https://modeso.atlassian.net/browse/LIMS-4914')
     def test027_update_material_type(self):
```

Comment:
this test case is not related to this PR so why you change it

Reply:
I made this branch as branch of update material_type test case as I needed functions in it, So any changes in  update material_type branch will be added here.